.github/workflows/ci.yml

```diff
@@ -0,0 +1,105 @@
+name: CI
+
+on:
+  push:
+  pull_request:
+  schedule:
+  - cron: "0 0 * * *"
+
+env:
+  - APPLICATION_ENV=development
+  - APPLICATION_STORE=DE
+
+jobs:
+  validation:
+    runs-on: ubuntu-18.04
+    strategy:
+      fail-fast: false
+      matrix:
+        php-version: [
+            '7.4'
```

Comment:
always min/max

Reply:
Maybe i understand it wrong, but for 7.3 we have separate "prefer-lowest" no?

Reply:
No, prefer lowest uses the same min, but additionally also min dependencies (not max dependencies).